spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.describe "The Guest", type: :feature do
+  before(:each) do
+    test_data
+  end
+  describe "#show page" do
+    it "displays the guests name and all info about the rooms stayed in" do
+      @billy.rooms.push(@motel_8_room_1)
+      @billy.rooms.push(@daysinn_room_5)
+
+      visit "/guests/#{@billy.id}"
+
+      expect(page).to have_content(@billy.name)
+
+      expect(page).to have_content(@billy.rooms.first.hotel.name)
```

Comment:
`within` blocks would help make these assertions more robust, but at the very least, it's good to include the fixed text around the dynamic values to make sure the value is in the right place, i.e. `have_content("Room Rate: #{room.rate}")`